### PR TITLE
[release-4.10] OCPBUGS-14279: Updates to use Java 11 by default

### PR DESCRIPTION
**Pull request by coreydaley**

### CODE REVIEW

File: 2/Dockerfile.rhel8 [hunk 1]
```diff
@@ -62,12 +62,18 @@ EXPOSE 8080 50000
 # /usr/lib64/jenkins will subsequently get redirected to /usr/lib/jenkins; it is confirmed that the 3.7 jenkins RHEL images
 # do *NOT* have a /usr/lib64/jenkins path
 RUN ln -s /usr/lib/jenkins /usr/lib64/jenkins && \
-    INSTALL_PKGS="dejavu-sans-fonts wget rsync gettext git tar zip unzip openssl bzip2 java-11-openjdk java-11-openjdk-devel java-1.8.0-openjdk java-1.8.0-openjdk-devel jq glibc-locale-source" && \
+    INSTALL_PKGS="dejavu-sans-fonts wget rsync gettext git git-lfs tar zip unzip openssl bzip2 java-11-openjdk java-11-openjdk-devel java-1.8.0-openjdk java-1.8.0-openjdk-devel jq glibc-locale-source" && \
```
Comment (apoorvajagtap) [inline 2/Dockerfile.rhel8:65]: I could see some reverts & restores around the use of ` xmlstarlet` in release-4.11+ branches. So, just checking here, if we need that for 4.10 anymore?

File: 2/Dockerfile.rhel8
Comment (apoorvajagtap) [inline 2/Dockerfile.rhel8:1]: just a typo of `"` at [Line54&55](https://github.com/openshift/jenkins/blob/2ce1911113c882671843eb032740f9f1ca51670e/2/Dockerfile.rhel8#L54-L55)

Comment (coreydaley) [reply 2/Dockerfile.rhel8:1]: fixing

**Comment by coreydaley** [comment]
@apoorvajagtap Comments addressed, ptal

**Comment by coreydaley** [comment]
/retest-required

level=error msg=Error: Error launching source instance: InsufficientInstanceCapacity: We currently do not have sufficient m6i.xlarge capacity in the Availability Zone you requested (us-west-1b). Our system will be working on provisioning additional capacity. You can currently get m6i.xlarge capacity by not specifying an Availability Zone in your request or choosing us-west-1a.

**Pull request merged**